db/schema.rb

```diff
@@ -199,8 +199,6 @@
   create_table "groups", force: :cascade do |t|
     t.string "name"
     t.text "description"
-    t.string "phone_number"
```

Comment:
@skng5  these keep coming up in your PRs. Why?

Reply:
@bibliotechy I may have to start with a fresh working repo.